docs/modules/ROOT/pages/how-tos/debug-nodes.adoc

```diff
@@ -1,6 +1,9 @@
 = Access nodes for debugging
 
-You can access nodes for debugging with `oc debug node` as long as the Kubernetes API is available.
+== Healthy nodes
+
+You can access nodes for debugging with `oc debug node` as long as the Kubernetes API is available and pods can be scheduled on the node.
+If this isn't the case see the next section.
```

Comment:
```suggestion
If this isn't the case see the <<Unhealthy nodes>>.
```

Please check with preview if this link actually works. Otherwise, check https://docs.asciidoctor.org/asciidoc/latest/macros/xref/ for alternatives.